reference/configuration.rst

```diff
@@ -33,6 +33,17 @@ Citus tracks worker nodes' locations and their membership in a shared hash table
 Data Loading
 ---------------------------
 
+citus.multi_shard_commit_protocol (enum)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+Sets the commit protocol to use when performing COPY on a hash distributed table. On each individual shard placement, the COPY is performed in a transaction block to ensure that no data is ingested if an error occurs during the COPY. However, there is a particular failure case in which the COPY succeeds on all placements, but a (hardware) failure occurs before all transactions commit. This parameter can be used to prevent data loss in that case by choosing between the following commit protocols: 
```

Comment:
The COPY will report success with warnings. The problem is that you cannot easily recover because it only failed for particular hash ranges.